src/resources/PDF-healthHygiene/main/styles.scss

```diff
@@ -0,0 +1,174 @@
+@media print {
```

Comment:
@asifamingov  can you review 

@ganapavz  is this ok for the swe? should we look at incorporating this into our swe?